### DISCUSSION TITLE
"Require annotated tags" tag protection

### DISCUSSION THREAD

**Discussion by lengau** [community/community#77443] [Repositories]
### Select Topic Area

Product Feedback

### Body

Tag rulesets should allow us to require that tags which meet a specific pattern be annotated. This is useful since release tags should be annotated, so blocking unannotated tags that look like a version tag would prevent lightweight tags from being included in a release.

**Comment by spenserblack** [comment]
Great idea! This can be especially helpful if you like to generate release notes from a tag's annotations.

**Comment by Loki-Afro** [comment]
is there any known alternative searching for a viable way for a while ever now and then ..

**Comment by AndersNestech** [comment]
Still a great idea. Are there any indications as to whether and when this will be implemented?
